src/components/IGConfigForm.jsx

```diff
@@ -0,0 +1,118 @@
+                  helpText="The unique identifier for the IG. Default: fhir.example"
+                />
+
+                <ReactFinalForm.Field
+                  required
+                  name="name"
+                  label="Name"
+                  component={InputFieldFF}
+                  className={styles.inputField}
+                  validate={hasValue}
+                  helpText="Default: ExampleIG"
+                />
+
+                <ReactFinalForm.Field
+                  required
+                  name="canonical"
+                  label="Canonical"
+                  component={InputFieldFF}
+                  className={styles.inputField}
+                  validate={hasValue}
+                  helpText="Default: http://example.org"
+                />
+
+                <ReactFinalForm.Field
+                  required
+                  name="status"
+                  label="Status"
+                  component={SingleSelectFieldFF}
+                  className={styles.inputField}
+                  validate={hasValue}
+                  helpText="Default: draft"
+                  options={[
+                    { label: 'draft', value: 'draft' },
+                    { label: 'active', value: 'active' },
+                    { label: 'retired', value: 'retired' },
+                    { label: 'unknown', value: 'unknown' },
+                  ]}
+                />
+
+                <ReactFinalForm.Field
+                  required
+                  name="version"
+                  label="Version"
+                  component={InputFieldFF}
+                  className={styles.inputField}
+                  validate={hasValue}
+                  helpText="Default: 0.1.0"
+                />
+
+                <ReactFinalForm.Field
+                  required
+                  name="releaseLabel"
+                  label="Release Label"
+                  component={InputFieldFF}
+                  className={styles.inputField}
+                  validate={hasValue}
+                  helpText="Default: ci-build"
+                />
+
+                <ReactFinalForm.Field
+                  required
+                  name="publisher.name"
+                  label="Publisher"
+                  component={InputFieldFF}
+                  className={styles.inputField}
+                  validate={hasValue}
+                  helpText="Default: DHIS2"
+                />
+
+                <ReactFinalForm.Field
+                  required
+                  name="publisher.url"
+                  label="Publisher URL"
+                  component={InputFieldFF}
+                  className={styles.inputField}
+                  validate={hasValue}
+                  helpText="Default: https://dhis2.org"
+                />
+              </div>
+              <div className={styles.buttonRow}>
+                <div className={styles.button}>
+                  <Button type="submit" secondary>
```

Comment:
(minor) would be great if this could be `disabled={!formIsValid}` so that there's a visual cue that `next` won't work if the form has errors